tests/fixtures/modules/import/error-strict.src.js

```diff
@@ -0,0 +1,5 @@
+import house from "house";
+
+with (house) {
+	console.log(roof);
+}
```

Comment:
Just to clarify, am I supposed to have a newline or new newline. Looking at the existing tests, i'm not seeing a consistent pattern. Most of the result files (generated) have no extra newline.